packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -27,66 +33,68 @@ const SettingsHeader = ( { sidebarName } ) => {
 		};
 	}, [] );
 
-	const [ documentAriaLabel, documentActiveClass ] =
+	const [ documentAriaLabel ] =
 		sidebarName === 'edit-post/document'
 			? // translators: ARIA label for the Document sidebar tab, selected. %s: Document label.
 			  [ sprintf( __( '%s (selected)' ), documentLabel ), 'is-active' ]
 			: [ documentLabel, '' ];
 
-	const [ blockAriaLabel, blockActiveClass ] =
+	const [ blockAriaLabel ] =
 		sidebarName === 'edit-post/block'
 			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
 			  [ __( 'Block (selected)' ), 'is-active' ]
 			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
 			  [ __( 'Block' ), '' ];
 
-	const [ templateAriaLabel, templateActiveClass ] =
+	const [ templateAriaLabel ] =
 		sidebarName === 'edit-post/document'
 			? [ __( 'Template (selected)' ), 'is-active' ]
 			: [ __( 'Template' ), '' ];
 
-	/* Use a list so screen readers will announce how many tabs there are. */
 	return (
-		<ul>
-			{ ! isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ documentActiveClass }` }
-						aria-label={ documentAriaLabel }
-						data-label={ documentLabel }
-					>
-						{ documentLabel }
-					</Button>
-				</li>
-			) }
-			{ isTemplateMode && (
-				<li>
-					<Button
-						onClick={ openDocumentSettings }
-						className={ `edit-post-sidebar__panel-tab ${ templateActiveClass }` }
-						aria-label={ templateAriaLabel }
-						data-label={ __( 'Template' ) }
-					>
-						{ __( 'Template' ) }
-					</Button>
-				</li>
-			) }
-			<li>
-				<Button
-					onClick={ openBlockSettings }
-					className={ `edit-post-sidebar__panel-tab ${ blockActiveClass }` }
-					aria-label={ blockAriaLabel }
-					// translators: Data label for the Block Settings Sidebar tab.
-					data-label={ __( 'Block' ) }
+		<>
+			<Tabs.TabList>
+				<Tabs.Tab
+					id={ 'edit-post/document' }
+					render={
+						isTemplateMode ? (
+							<Button
+								onClick={ openDocumentSettings }
+								className={ `edit-post-sidebar__panel-tab` }
```

Comment:
Is this classname still necessary, now that the tab styles come directly from the `Tabs` component?

Reply:
Good point, I don't think it is. The only place this classname is in use that I can see is [part of a style override in a different sidebar.](https://github.com/WordPress/gutenberg/blob/12c17e509ce4d5bce3f7e811f283520499a24880/packages/edit-post/src/components/secondary-sidebar/style.scss#L36).